libs/common/src/index.ts

```diff
@@ -6,6 +6,5 @@ export * from './test-helpers';
 export * from './types';
 export * from './middleware';
 export * from './utils';
-export * from './constants';
```

Comment:
What's the reason for this please?

Reply:
I noticed it was repeated further up in the file

Reply:
Great spot!